pageserver/src/tenant/timeline/layer_manager.rs

```diff
@@ -33,6 +33,11 @@ impl LayerManager {
         }
     }
 
+    pub(crate) fn clear(&mut self) {
+        std::mem::take(&mut self.layer_map);
```

Comment:
maybe `self.layer_map = LayerMap::default()` is more clear here what it is doing.